tags/Coveo/CoveoAnalyticsLogEvent/Coveo Analytics Log Event.tpl

```diff
@@ -0,0 +1,446 @@
+        "valueValidators": [
+          {
+            "type": "NON_EMPTY"
+          }
+        ],
+        "displayName": "Content ID Key",
+        "defaultValue": "permanentid",
+        "simpleValueType": true,
+        "name": "contentIdKey",
+        "type": "TEXT"
+      },
+      {
+        "help": "The value of the field used to trace back the current page in the Coveo Index.",
+        "valueValidators": [
+          {
+            "type": "NON_EMPTY"
+          }
+        ],
+        "displayName": "Content ID Value",
+        "simpleValueType": true,
+        "name": "contentIdValue",
+        "type": "TEXT"
+      }
+    ]
+  },
+  {
+    "enablingConditions": [
+      {
+        "paramName": "eventType",
+        "type": "EQUALS",
+        "paramValue": "custom"
+      }
+    ],
+    "displayName": "Custom Event Metadata",
+    "name": "Custom Event Metadata",
+    "groupStyle": "ZIPPY_OPEN",
+    "type": "GROUP",
+    "subParams": [
+      {
+        "help": "",
+        "valueValidators": [
+          {
+            "type": "NON_EMPTY"
+          }
+        ],
+        "displayName": "Event Type",
+        "simpleValueType": true,
+        "name": "customEventType",
+        "type": "TEXT"
+      },
+      {
+        "help": "",
+        "valueValidators": [
+          {
+            "type": "NON_EMPTY"
+          }
+        ],
+        "displayName": "Event Value",
+        "simpleValueType": true,
+        "name": "customEventValue",
+        "type": "TEXT"
+      }
+    ]
+  },
+  {
+    "displayName": "Document Metadata",
+    "name": "Document Metadata",
+    "groupStyle": "ZIPPY_OPEN",
+    "type": "GROUP",
+    "subParams": [
+      {
+        "help": "The current language of the page's content. If not properly detected, will fallback on \"en\".",
+        "valueValidators": [
+          {
+            "type": "NON_EMPTY"
+          }
+        ],
+        "displayName": "Language",
+        "simpleValueType": true,
+        "name": "language",
+        "type": "TEXT"
+      },
+      {
+        "help": "(Optional) The current URL of the page. If not set, \"window.location\" will be used.",
+        "displayName": "Location",
+        "simpleValueType": true,
+        "name": "location",
+        "type": "TEXT"
+      },
+      {
+        "help": "(Optional) The title of the document. If not set, \"document.title\" will be used.",
+        "displayName": "Title",
+        "simpleValueType": true,
+        "name": "title",
+        "type": "TEXT"
+      },
+      {
+        "help": "(Optional) Boolean representing whether the current visit is from an anonymous user.",
+        "displayName": "Is Anonymous",
+        "simpleValueType": true,
+        "name": "isAnonymous",
+        "type": "TEXT"
+      }
+    ]
+  },
+  {
+    "type": "SIMPLE_TABLE",
+    "name": "customData",
+    "displayName": "Custom Data",
+    "simpleTableColumns": [
```

Comment:
The Simple Table is much nicer! However, would it be an even better UI to embed the Simple Table into a Group, like the previous fields, too? So create a group named "Custom Data", and then add the simple table as a nested field (without a Display Name, since the group title handles that)?